Sources/BraintreePayPal/RecurringBillingMetadata/BTPayPalRecurringBillingDetails.swift

```diff
@@ -12,8 +12,9 @@ public struct BTPayPalRecurringBillingDetails {
     private let productQuantity: Int?
     private let oneTimeFeeAmount: String?
     private let shippingAmount: String?
-    private let productAmount: String?
+    private let productPrice: String?
```

Comment:
Sounds like we want to revert this change and keep `productAmount` based on this comment: https://github.com/braintree/braintree_ios/pull/1373#discussion_r1688004735